protogen/gen/ocis/services/settings/v0/settings.pb.web.go

```diff
@@ -86,6 +86,10 @@ func (h *webBundleServiceHandler) ListBundles(w http.ResponseWriter, r *http.Req
 		return
 	}
 
+	// THIS WONT WORK - IT IS A AUTOGNERATED FILE
+	// HOW DO WE PASS THE LOCALE TO THE HANDLER?
+	req.Locale = r.Header.Get("Accept-Language")
```

Comment:
This won't work. It is a autogenerated file. This brings a problem: How the we pass the language to the handler? web could send it in the body of the request, but that doesn't seem right. Any other options?